### PR TITLE
Users settings

### PR DESCRIPTION
**Pull request by ArtyomKosakyan**
Added settings functionality for Admin,Passenger,Driver. They can change his phone_number, name and etc.
Added "active" and "verified" indicators for Driver.

### CODE REVIEW

**Review by iwazowsky — APPROVED** [review]
MIght need some refactoring to keep code DRY, but overall it's fine

File: src/components/Admin/ClientFieldPassenger.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ClientFieldPassenger = ({ passenger }) => {
+  const {
+    id, phoneNumber, firstName, lastName,
```
Comment (vnersisyan-vineti) [inline src/components/Admin/ClientFieldPassenger.js:6]: destruct in props please

File: src/components/Admin/ClientList.js [hunk 2]
```diff
@@ -1,48 +1,33 @@
...
-  ));
-
   return (
     <div>
-      <div className="menu">
-        <ul>
-          {buttons}
-        </ul>
-      </div>
+      {userId
```
Comment (vnersisyan-vineti) [inline src/components/Admin/ClientList.js:21]: userId && ()

File: src/components/Buttons.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ export const PageButton = ({
             className={className}
```
Comment (vnersisyan-vineti) [inline src/components/Buttons.js:20]: className can be not required props, and has default value `''`, in that case you can remove ternary operator

File: src/components/Client/Taxi.js [hunk 4]
```diff
@@ -15,6 +18,7 @@ const loader = new Loader({
 
 const Taxi = ({ appState }) => {
   const state = appState;
+  const { userId } = appState;
```
Comment (vnersisyan-vineti) [inline src/components/Client/Taxi.js:21]: destruct in props

File: src/components/Driver/DriverPage.js [hunk 5]
```diff
@@ -112,19 +112,38 @@ const DriverPage = ({ appState }) => {
...
+            menuButtons={DriverPageButtons}
+          />
+        ) : false}
+      <div className="ui-component container-md">
+        <h1>User functionality container</h1>
+        <div
+          ref={handleMap}
+          className="text-center border top-50 start-50 translate-middle mb-6 maps"
+        />
+        {showConfirm
```
Comment (vnersisyan-vineti) [inline src/components/Driver/DriverPage.js:130]: showConfirm && ()

File: src/components/Driver/DriverPage.js [hunk 6]
```diff
@@ -112,19 +112,38 @@ const DriverPage = ({ appState }) => {
   };
 
   return (
-    <div className="position-relative">
-      <div className="card text-center position-absolute top-0 start-0 ms-4 mt-3" style={{ width: '150px', height: '166px' }}>
-        <ul className="list-group list-group-flush">
-          {menuItems}
-        </ul>
+    <div>
+      {userId
```
Comment (vnersisyan-vineti) [inline src/components/Driver/DriverPage.js:116]: userId && ()

File: src/components/Driver/DriverProfile.js [hunk 7]
```diff
@@ -3,31 +3,57 @@ import PropTypes from 'prop-types';
...
+    }
   }, []);
 
   return (
-    <div className="ui-component container-md">
-      <div className="card text-center position-absolute top-50 start-50 translate-middle">
-        {state.isLoading ? <Loading /> : false}
-        <ProfileInfoTable fieldsData={state.user} />
+    <div>
+      {userId
```
Comment (vnersisyan-vineti) [inline src/components/Driver/DriverProfile.js:31]: userId && ()

File: src/components/Driver/DriverSettings.js [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+import SettingsHelper from '../../helpers/SettingsHelper';
+import { appRoutes, DriverPageButtons } from '../../utils/configs';
+import UserMenu from '../layouts/UserMenu';
+
+const DriverSettings = ({ match }) => {
+  const userId = match.params.id;
+
+  return (
+    <div>
+      {userId
```
Comment (vnersisyan-vineti) [inline src/components/Driver/DriverSettings.js:14]: userId && ()

File: src/components/Driver/DriverProfile.js [hunk 9]
```diff
@@ -3,31 +3,57 @@ import PropTypes from 'prop-types';
 import ProfileInfoTable from '../../utils/ProfileInfoTable';
 import Loading from '../../shared/Animations/Loading';
 import getUserData from '../../helpers/ProfilePageHelper';
+import { DriverPageButtons, appRoutes } from '../../utils/configs';
+import UserMenu from '../layouts/UserMenu';
 
 const DriverProfile = ({ match }) => {
   const [state, setState] = useState({
     userId: match.params.id,
```
Comment (vnersisyan-vineti) [inline src/components/Driver/DriverProfile.js]: destruct in props

File: src/components/Driver/DriverSettings.js [hunk 10]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { mapStateToProps } from '../../redux/actions';
+import SettingsHelper from '../../helpers/SettingsHelper';
+import { appRoutes, DriverPageButtons } from '../../utils/configs';
+import UserMenu from '../layouts/UserMenu';
+
+const DriverSettings = ({ match }) => {
+  const userId = match.params.id;
```
Comment (vnersisyan-vineti) [inline src/components/Driver/DriverSettings.js]: destruct in props

File: src/components/RegistrationForm.js [hunk 11]
```diff
@@ -20,71 +23,86 @@ const RegistrationForm = ({ data, header, children }) => {
...
+                className="form-control"
+              />
+            </label>
+          </div>
+        )
+        : <></>}
       { children }
     </div>
   );
 };
```
Comment (vnersisyan-vineti) [inline src/components/RegistrationForm.js:94]: put empty line

File: src/utils/ProfileSettingsFields.js [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+  const fields = Object.entries(fieldsData).map((key) => (
+    <table key={key[0]} className="table table-borderless">
+      <thead>
+        <tr>
+          <th>{ key[0] }</th>
+          <th>{typeof key[1] === 'boolean' ? key[1].toString() : key[1]}</th>
+        </tr>
+      </thead>
+    </table>
+  ));
```
Comment (vnersisyan-vineti) [inline src/utils/ProfileSettingsFields.js:13]: empty line here

**Review by vnersisyan-vineti — CHANGES_REQUESTED** [review]
minor comments

File: src/components/Client/Taxi.js (hunk 4 above)
Comment (ArtyomKosakyan) [reply src/components/Client/Taxi.js:21]: I used appState too

**Review by vnersisyan-vineti — APPROVED** [review]
Awesome

**Pull request merged**